tests/features/common/kie-common.feature

```diff
@@ -124,4 +124,37 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_LOGIN_MODULE | required  |
     Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="LdapExtended" flag="required">
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
+    Then container log should contain Kie Server's cluster for JBPM fail over enabled
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <cache-container name="jbpm">
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <transport lock-timeout="60000"/>
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <replicated-cache name="nodes">
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <transaction mode="BATCH"/>
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain </replicated-cache>
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <replicated-cache name="jobs">
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <transaction mode="BATCH"/>
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain </replicated-cache>
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should contain </cache-container>
+    And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value 60000 on XPath //*[local-name()='cache-container']/*[local-name()='jbpm']/*[local-name()='transport lock-timeout']
```

Comment:
please use the same pattern for the conditions,  use XPath for text.

Reply:
maybe it is better to use XPath on all conditions.

Reply:
Removed xpath pattern

Reply:
Done